docs/guides/compose.mdx

```diff
@@ -0,0 +1,292 @@
+    cache: null
+```
+
+</TabItem>
+<TabItem value="json" label="json">
+```json showLineNumbers
+{
+  "server": {
+    "graphiql": true,
+    "hostname": "0.0.0.0",
+    "port": 8000
+  },
+  "upstream": {
+    "baseURL": "http://jsonplaceholder.typicode.com",
+    "httpCache": true,
+    "batch": {
+      "maxSize": 100,
+      "delay": 100,
+      "headers": []
+    }
+  },
+  "schema": {
+    "query": "Query"
+  },
+  "types": {
+    "Post": {
+      "fields": {
+        "body": {
+          "type": "String",
+          "required": true,
+          "cache": null
+        },
+        "id": {
+          "type": "Int",
+          "required": true,
+          "cache": null
+        },
+        "title": {
+          "type": "String",
+          "required": true,
+          "cache": null
+        },
+        "user": {
+          "type": "User",
+          "cache": null
+        },
+        "userId": {
+          "type": "Int",
+          "required": true,
+          "cache": null
+        }
+      },
+      "cache": null
+    },
+    "Query": {
+      "fields": {
+        "posts": {
+          "type": "Post",
+          "list": true,
+          "http": {
+            "path": "/posts"
+          },
+          "cache": null
+        },
+        "user": {
+          "type": "User",
+          "args": {
+            "id": {
+              "type": "Int",
+              "required": true
+            }
+          },
+          "http": {
+            "path": "/users/{{args.id}}"
+          },
+          "cache": null
+        },
+        "users": {
+          "type": "User",
+          "list": true,
+          "http": {
+            "path": "/users"
+          },
+          "cache": null
+        }
+      },
+      "cache": null
+    },
+    "User": {
+      "fields": {
+        "email": {
+          "type": "String",
+          "required": true,
+          "cache": null
+        },
+        "id": {
+          "type": "Int",
+          "required": true,
+          "cache": null
+        },
+        "name": {
+          "type": "String",
+          "required": true,
+          "cache": null
+        },
+        "phone": {
+          "type": "String",
+          "cache": null
+        },
+        "username": {
+          "type": "String",
+          "required": true,
+          "cache": null
+        },
+        "website": {
+          "type": "String",
+          "cache": null
+        }
+      },
+      "cache": null
+    }
+  }
+}
+```
+</TabItem>
+</Tabs>
+
+### Enhanced Editor Support
+
+To leverage autocomplete and validation for Tailcall configurations, utilize `.tailcallrc.graphql` for GraphQL configurations and `.tailcallrc.schema.json` for JSON and YAML configurations. These files enhance editor support by providing schema definitions, facilitating faster and error-free configuration.
+
+For more detailed information on configuring TailCall for enhanced editor support, refer to tailcall's [init command](/docs/guides/cli.md#init).
```

Comment:
@neo773 I think we should expand a little bit more on how to use these files. For example how can I setup `.tailcallrc.graphql` with `.graphqlrc.yml` and use `.tailcallrc.schema.json` with a json file. A step by step guide would be useful here.